Header header

# TODO: Add a pose here for debugging
geometry_msgs/Pose pose

# Color image of the scene
sensor_msgs/Image color_data
# uint8[] color_data

# Depth image of the scene
sensor_msgs/Image depth_data
# uint8[] depth_data
